File: programmingclub/abc/abc.java
import java.io.*;
import java.util.*;

public class abc {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        int[] nums = new int[7];
        for (int i = 0; i < 7; i++) {
            nums[i] = in.nextInt();
        }

        Arrays.sort(nums);

        int a = nums[0];
        int b = nums[1];
        int c = nums[6] - a - b;

        System.out.println(a + " " + b + " " + c);
    }
}

File: programmingclub/abc/abc.cpp
#include <bits/stdc++.h>
using namespace std;

int nums[7];

int main() {
    for (int& num : nums) {
        cin >> num;
    }

    sort(nums, nums + 7);

    int a = nums[0];
    int b = nums[1];
    int c = nums[6] - a - b;

    cout << a << " " << b << " " << c << endl;

    return 0;
}

